src/kmod/file-write-cache.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2016-2020 VMware, Inc.
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+#include "file-write-cache.h"
+#include <linux/jhash.h>
+#include <linux/list.h>
+#include <linux/spinlock.h>
+
+struct fwc_bkt {
+	spinlock_t	 lock;
+	u32		 size;
+	struct list_head list;
+};
+
+struct fwc_key {
+	pid_t tgid;
+	ino_t inode;
+	dev_t dev;
+	u64   time;
+};
+
+struct fwc_entry {
+	u32		 hash;
+	struct fwc_key	 key;
+	struct list_head list;
+	u32		 hits;
```

Comment:
Any plan to use/expose the "hits" field?  Seems the only operation is to increase by one.

Reply:
It will be useful for future customized eviction. In a core dump we can dump this table or entries and look at hit counts.